File: app/src/main/java/cc/smartcash/wallet/Services/SAPIService.java
package cc.smartcash.wallet.Services;


import cc.smartcash.wallet.Models.SapiAddressBalance;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface SAPIService {

    @GET("address/balance/{address}")
    Call<SapiAddressBalance> GetAddressBalance(@Path("address") String address);


}

File: app/src/main/java/cc/smartcash/wallet/Services/SAPIService.kt
package cc.smartcash.wallet.Services


import cc.smartcash.wallet.Models.SapiAddressBalance
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface SAPIService {

    @GET("address/balance/{address}")
    fun getAddressBalance(@Path("address") address: String): Call<SapiAddressBalance>


}